spec/predicates/runtime-trace.md

```diff
@@ -0,0 +1,2677 @@
+# Predicate type: Runtime Trace
+
+Type URI: https://in-toto.io/runtime-trace/v0.1.0
+
+Version: 0.1.0
+
+## Purpose
+
+This predicate is used to describe system events that were part of the artifact build process. Artifact build processes today are complex and opaque with little to no insights into build actions. However, there are emerging introspection techniques that now allow us to monitor and capture system event logs during the build.
+
+## Prerequisites
+
+Understanding of monitoring tools and the larger in-toto attestation framework.
+
+## Model
+
+The structure of this predicate closely mirrors the structure of the SLSA Provenance attestation as they are complementary. `monitor` identifies the specific instance of a tool that’s observing the build, while `build` contains information that when put together uniquely identifies the exact instance of the job being observed. `invocation` details the configurations of `monitor` and the `monitorLog` contains the actual runtime trace information.
```

Comment:
I'll go a step further and say that it should probably not refer to SLSA Provenance at all, and instead define its fields in a standalone manner. This may be something that goes into ITE-9 as well so we don't have issues with specs going out of sync with each other over time. It's hard to ensure references are updated otherwise. Thoughts?

Reply:
I took a pass at removing those mentions. I think it's fine to be inspired by another attestation type but we should refrain from referring to other specs unless updates to both are closely linked to each other.